src/RawPOSIX/src/safeposix/syscalls/fs_calls.rs

```diff
@@ -1903,8 +1903,25 @@ impl Cage {
     }
 }
 
+/// Lind-WASM is running as same Linux-Process from host kernel perspective, so standard fds shouldn't 
+/// be closed in Lind-WASM execution, which preventing issues where other threads might reassign these 
+/// fds, causing unintended behavior or errors. 
 pub fn kernel_close(fdentry: fdtables::FDTableEntry, _count: u64) {
-    let _ret = unsafe {
+    let kernel_fd = fdentry.underfd as i32;
+
+    // TODO:
+    // Need to update once we merge with vmmap-alice
```

Comment:
Ah I see why this was confusing, the constants changes are still only on the vmmap branch. This is fine for now.